app/containers/Forms/Form.tsx

```diff
@@ -149,40 +150,7 @@ export const FormComponent: React.FunctionComponent<Props> = ({
         </div>
       </JsonSchemaForm>
       <style jsx global>
-        {`
-          .rjsf .form-row {
-            margin: 20px 0 20px;
-          }
-          .emission-form .rjsf .form-row {
-            margin: 0;
-          }
-          .no-col-padding .form-row > .col,
-          .no-col-padding .form-row > [class*='col-'] {
-            padding: 0;
-          }
-          .form-submit {
-            border: 1px solid #bbb;
-            padding: 30px;
-            background: #eee;
-            border-radius: 6px;
-          }
-          .errors {
-            margin-left: 20px;
-            padding: 20px;
-            background: #ce5c5c;
-            color: white;
-            font-size: 20px;
-          }
-          ul.error-detail {
-            padding: 0;
-            list-style: none;
-          }
-          ul.error-detail li.text-danger {
-            background: #ce5c5c;
-            color: white !important;
-            padding: 5px;
-          }
-        `}
+        {globalFormStyles}
```

Comment:
These have been extracted into a [style module](https://github.com/bcgov/cas-ciip-portal/pull/565/files#diff-fb2e8a7bb50109eb53c0bcc780d27259) in order to share the form styles between the industry reporter form and other forms (in this case, the `/admin/reporting-years` form).

Reply:
feel free to update the snapshots ( `yarn test -u`)

Reply:
PS: Here's the [link to the docs](https://github.com/zeit/styled-jsx#external-css-and-styles-outside-of-the-component) that you were wondering about for creating a style module.